clang/tools/IndexStore/CMakeLists.txt

```diff
@@ -28,11 +23,10 @@ add_clang_library(IndexStore SHARED
   ${SOURCES}
 
   LINK_LIBS
-  ${LIBS}
+  clangIndex
+  clangIndexDataStore
 
   LINK_COMPONENTS
-  ${LLVM_TARGETS_TO_BUILD}
```

Comment:
This is also in `libclang` which seems... very strange. I'm not sure why this would be needed and it's been here since the index store was added.